src/common/src/telemetry/report.rs

```diff
@@ -91,9 +100,29 @@ where
                 )
             });
 
+        let (tx, mut event_rx) = tokio::sync::mpsc::unbounded_channel::<PbEventMessage>();
+        TELEMETRY_EVENT_REPORT_TX.set(tx).unwrap_or_else(|_| {
+            tracing::warn!(
+                "Telemetry failed to set event reporting tx, event reporting will be disabled"
+            );
+        });
+        let mut event_stash = Vec::new();
+
         loop {
             tokio::select! {
                 _ = interval.tick() => {},
+                event = event_rx.recv() => {
+                    debug_assert!(event.is_some());
+                    event_stash.push(event.unwrap());
```

Comment:
```thread 'rw-standalone-frontend' panicked at /risingwave/src/common/src/telemetry/report.rs:116:44:
called `Option::unwrap()` on a `None` value
```

when using standalone, each of the components panics

Reply:
I guess no, will validate tmr.